Heber/src/main/java/springApp/java/com/example/demo/controllers/InstrumentoController.java

```diff
@@ -0,0 +1,156 @@
+package springApp.java.com.example.demo.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import springApp.java.com.example.demo.exceptions.InstrumentoNoEncontradoException;
+import springApp.java.com.example.demo.models.AccionModel;
+import springApp.java.com.example.demo.models.BonoModel;
+import springApp.java.com.example.demo.models.InstrumentoFinancieroModel;
+import springApp.java.com.example.demo.services.InstrumentoService;
+import springApp.java.com.example.demo.exceptions.InstrumentoDuplicadoException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/instrumentos") // Este es el path base del controller
+public class InstrumentoController {
+
+    @Autowired
+    InstrumentoService instrumentoService;
+
+    @GetMapping("/hola")
+    public String instrumento() {
+        return "asdasd";
+    }
+
+    // GET URL: localhost:8081/service-service
+    @RequestMapping("/hello-service/{id}")
+    public String helloService(@PathVariable("id") Long id) {
+        System.out.println("el id es: " + id);
+        return instrumentoService.helloService();
+    }
+
+
+    // GET URL: localhost:8081/hello-repository
+    @RequestMapping("/hello-repository")
+    public String helloRepository() {
+        return instrumentoService.helloRepository();
+    }
```

Comment:
Se que son metodos de pruebas, cuando puedas eliminalos si no se usan y si es para tener un endpoint de pruebas, lo mas comun es que se cree un endpoint de tipo get /pong que devuelve un texto de ping

```suggestion

    @GetMapping("/pong")
    public String Pong() {
        return "Ping";
    }
```